usaspending_api/database_scripts/matview_generator/tas_award_matview.json

```diff
@@ -0,0 +1,64 @@
+{
+  "final_name": "tas_award_matview",
+  "refresh": true,
+  "matview_sql": [
+    "select distinct",
+    "    md5(concat_ws(",
+    "        '|',",
```

Comment:
What benefit does the pipe character  provide when creating the MD5 UUID?

Reply:
You mean in comparison to some other character or no character at all?  It's no more or less valuable than a space in this particular case.  I'd prefer to use some kind of separator character to minimize the chance of collisions.

Reply:
By way of totally contrived example, the following two values would hash to the same thing without some sort of delimiter (preferably one that "cannot" exist in the actual data):

| aid | a | main | sub | transaction_id |
|-----|---|-------|-----|-----------------|
| 011  |  X | 1234  |       |  12345678         |
| 011  |  X | 1234  | 123  |  45678              |

Reply:
Thanks for the example, it makes complete sense to me.